src/native/corehost/fxr/fx_resolver.cpp

```diff
@@ -285,6 +288,58 @@ namespace
 
         return new fx_definition_t(fx_ref.get_fx_name(), selected_fx_dir, oldest_requested_version, selected_fx_version);
     }
+
+#ifdef TARGET_OSX
+    bool isRunningOnArm64()
```

Comment:
Nit: the naming convention should call this something like `is_running_on_arm64` (I don't like it, but for consistency).